apps/backend/convex/entities/types.ts

```
@@ -22,6 +22,7 @@ export type ThreadWithDetails = Infer<typeof ThreadWithDetailsReturn>
 export type User = Infer<typeof UserReturn>
 
 export type MessageRoles = Infer<typeof MessageRoles>
+export type ModelParameters = Infer<AsObjectValidator<typeof ModelParameters>>
```

Biome

[error] Shouldn't redeclare 'ModelParameters'. Consider to delete it or rename it.

'ModelParameters' is defined here:


(lint/suspicious/noRedeclare)